setup.py

```diff
@@ -15,6 +15,12 @@
         "pydantic==2.9.2",
         "pydantic-core==2.23.4",
     ],
+    ## Has to be installed as: pip install aimon[aimon_llamaindex]
+    extras_require={
+        "aimon_llamaindex":[
```

Comment:
should be aimon-llamaindex v/s underscore

Reply:
Dependency naming changed to: aimon-llamaindex
Module name kept as aimon_llamaindex, and setup.py modified with:
`    
packages=find_packages(exclude=['aimon_llamaindex', 'aimon_llamaindex.*']),
## To exclude this aimon_llamaindex with the default installation

extras_require={
        "aimon-llamaindex":[
        "llama-index",
        "aimon_llamaindex"
        ],
    },`